nacc_attribute_deriver/attributes/base/uds_namespace.py

```diff
@@ -0,0 +1,83 @@
+"""Class to define UDS-specific attributes."""
+
+from datetime import date, datetime
+from typing import Any, Optional
+
+from nacc_attribute_deriver.attributes.base.namespace import (
+    FormNamespace,
+    SubjectDerivedNamespace,
+)
+from nacc_attribute_deriver.schema.errors import MissingRequiredError
+from nacc_attribute_deriver.symbol_table import SymbolTable
+
+
+class UDSNamespace(FormNamespace):
+    def __init__(self, table: SymbolTable) -> None:
+        """Check that this is a UDS form."""
+        super().__init__(table)
+
+        module = self.get_value("module")
+        if not module or module.upper() != "UDS":
+            raise MissingRequiredError("Current file is not an UDS form")
+
+        self.__subject_derived = SubjectDerivedNamespace(table)
```

Comment:
I feel like this and the associated methods are in the wrong place. 

The UDS namespace is a refinement of a form-namespace, so it doesn't make sense to include the subject in this class